src/ngraph/runtime/cpu/mkldnn_utils.cpp

```diff
@@ -1755,8 +1755,10 @@ bool runtime::cpu::mkldnn_utils::is_bf16_supported()
     }
     catch (const mkldnn::error& e)
     {
+        std::cout << "bf16 is not supported: " << e.message << std::endl;
```

Comment:
use NGRAPH_ERR

Reply:
I will revert this commit. This is for debugging mac.